### PR TITLE
add func and tests for task 9

### PR DESCRIPTION
**Pull request by vh232**

### CODE REVIEW

File: models/models.js [hunk 1]
```diff
@@ -68,4 +68,22 @@ exports.postNewComment = (inputId, comment) => {
...
+        }
+        return db.query(`SELECT * FROM comments WHERE comment_id = $1;`, id)
+    })
+    .then(({ rows }) => {
+        if (!rows.length) {
+            return rows
+        } else {
+            return 'unable to delete comment'
+        }
+    })
```
Comment (aloush-dev) [inline models/models.js:103]: This portion of the model isn't really needed.
Once we have a respond and we know the comment has been deleted we're all good.

**Review by aloush-dev — APPROVED** [review]
Awesome!
Merge when ready.

**Review by aloush-dev — APPROVED** [review]
Awesome!
Merge when ready.

File: models/models.js (hunk 1 above)
Comment (vh232) [reply models/models.js:103]: Sweet thank you

**Pull request merged**